### ISSUE TITLE
[Feature request]: Specify database schema when using PostgREST

### ISSUE THREAD

**Issue by profiluefter** [supabase-community/supabase-kt#50]
### General Info

- [X] I installed the latest version of Supabase Kt
- [X] I checked for similar feature requests

### Feature request

Is there an intended way to specify a database schema when using the PostgREST client? The wiki describes how to do it using Realtime however I couldn't find a proper solution for REST.

A possible solution/workaround is setting the request headers manually:

```kt
        createSupabaseClient(supabaseUrl, supabaseKey) {
            install(Postgrest)
            install(Realtime)
            httpConfig {
                defaultRequest {
                    header("Accept-Profile", "api")
                    header("Content-Profile", "api")
                }
            }
        }
```

It would be nice to be able to specify the schema in the client (like with supabase-js) and maybe override it per request (common feature request for supabase-js last time i checked)

### Usecase

If you use any other database schema than public you would need this feature (or the workaround above). Different schemas are currently already supported for Realtime so it makes sense to also support it for REST.

**Comment by jan-tennert** [comment]
Thinking about possible syntax. May this be a solution?
```kotlin
val client = createSupabaseClient(url, key) {
    install(Postgrest) { 
        defaultSchema = "api" //defaults to public when not overridden 
    }
}

//then in a request maybe
client.postgrest["users", "public"]
    ...
```

**Comment by profiluefter** [comment]
looks good to me!

**Comment by jan-tennert** [comment]
Swapped the arguments:
```kotlin
client.postgrest["public", "users"]
```

**Comment by jan-tennert** [comment]
Included in [0.8.0-alpha-1](https://github.com/supabase-community/supabase-kt/releases/tag/0.8.0-alpha-1), let me know if there are any issues!

**Comment by profiluefter** [comment]
Just tried it and it works! Thanks for implementing this so fast!
